OpenRA.Mods.Common/Traits/World/PaletteFromGimpOrJascFile.cs

```diff
@@ -45,7 +46,68 @@ class PaletteFromGimpOrJascFileInfo : ITraitInfo
 		[Desc("Index set to be fully transparent/invisible.")]
 		public readonly int TransparentIndex = 0;
 
+		[Desc("Whether this palette is available for cursors.")]
+		public readonly bool CursorPalette = false;
+
 		public object Create(ActorInitializer init) { return new PaletteFromGimpOrJascFile(init.World, this); }
+
+		string IProvidesCursorPaletteInfo.Palette { get { return CursorPalette ? Name : null; } }
+
+		ImmutablePalette IProvidesCursorPaletteInfo.ReadPalette(IReadOnlyFileSystem fileSystem)
+		{
+			var s = fileSystem.Open(Filename);
```

Comment:
Is there a reason you dropped the `using` from this?

Reply:
That was an accident. Added it back.